code/game/area/gelida_iv.dm

```diff
@@ -22,46 +22,6 @@
 	name = "gelida - Out Of Bounds"
 	icon_state = "unknown"
 
-//Landing Zone 1
-
-/area/gelida/landing_zone_1
-	name = "Landing Zone One"
-	icon_state = "explored"
-	outside = FALSE
-	minimap_color = MINIMAP_AREA_LZ
-
-/area/shuttle/drop1/gelida
-	name = "Gelida IV - Dropship Alamo Landing Zone"
-	icon_state = "away1"
-	outside = FALSE
-	minimap_color = MINIMAP_AREA_LZ
-
-/area/gelida/landing_zone_1/lz1_console
-	name = "Gelida IV - Dropship Alamo Console"
-	icon_state = "tcomsatcham"
-	requires_power = FALSE
-	outside = FALSE
-	minimap_color = MINIMAP_AREA_LZ
-
-//Landing Zone 2
-
-/area/gelida/landing_zone_2
-	name = "Gelida IV - Landing Zone Two"
-	icon_state = "explored"
-	outside = FALSE
```

Comment:
whats going on with all the lz removal & pathing changes

Reply:
The code for determining if a shuttle landed outside an LZ now just checks if the area the shuttle landed in is a type of `/landing_zone/`, hence the repaths.